src/print-1-10-growing.py

```diff
@@ -1,2 +1,7 @@
 
 # Print the numbers described in the exercise
+x = ''
+for number in range (1,11,1):
+    x += '{}'.format(number)
+    print(x.strip())
```

Comment:
The way you construct `x`, there shouldn't be any leading or trailing spaces, so `x.strip()` doesn't do anything.

You might want to have spaces between the numbers, but that is a separate issue (which might be what you were trying to solve with formatting strings?)